src/benchmarks/gc/src/exec/GCPerfSim/GCPerfSim.cs

```diff
@@ -2499,27 +2476,13 @@ bool GoToNextPhase()
     // Allocates object and survives it. Returns the object size.
     void MakeObjectAndMaybeSurvive()
     {
-        // if (isLarge && args.lohPauseMeasure)
```

Comment:
For example, the top comment describes `-lohPauseMeasure`, which isn't actually parsed.  Would this be part of it?

Reply:
we should not get rid of this code - it's useful for measuring LOH alloc pauses. see the comment at the beginning in "Other things worth noting" -

> 1) There's also a lohPauseMeasure var that you could make into a config - for now I just always measure pauses for LOH 
>    allocs (since it was used for measuring LOH alloc pause wrt BGC sweep).

if you want to improve this, you could make it into a config.

Reply:
Sure, I'll add a comment with this code as a TODO and reference the comment.